code/_onclick/ai.dm

```diff
@@ -46,14 +46,6 @@
 	if(isnull(pixel_turf))
 		return
 	if(!can_see(A))
-		if(isturf(A)) //On unmodified clients clicking the static overlay clicks the turf underneath
-			return //So there's no point messaging admins
-		message_admins("[ADMIN_LOOKUPFLW(src)] might be running a modified client! (failed can_see on AI click of [A] (Turf Loc: [ADMIN_VERBOSEJMP(pixel_turf)]))")
-		var/message = "[key_name(src)] might be running a modified client! (failed can_see on AI click of [A] (Turf Loc: [AREACOORD(pixel_turf)]))"
-		log_admin(message)
-		if(REALTIMEOFDAY >= chnotify + 9000)
-			chnotify = REALTIMEOFDAY
```

Comment:
Could you kill this variable while you're here, `chnotify`? Yes, this is its one use.

Reply:
(it's in `code\modules\mob\living\silicon\ai\ai.dm`)

Reply:
we have too many open prs so im just gonna merge this anyway u can web edit it later